src/jit/emitxarch.cpp

```diff
@@ -9706,7 +9781,7 @@ BYTE* emitter::emitOutputRRR(BYTE* dst, instrDesc* id)
 
     instruction ins = id->idIns();
     assert(IsAVXInstruction(ins));
-    assert(IsThreeOperandAVXInstruction(ins));
+    assert(IsThreeOperandAVXInstruction(ins) || ins == INS_vblendvps || ins == INS_vblendvpd || ins == INS_vpblendvb);
```

Comment:
I think it would be useful to have the `ins == INS_vblendvps || ins == INS_vblendvpd || ins == INS_vpblendvb` logic wrapped in a helper method.

Reply:
Ok, will do.